--- a/debian/rules.d/Makefile.inc	2025-10-20 14:02:29.839019864 +0200
+++ b/debian/rules.d/Makefile.inc	2025-10-20 14:56:53.299032533 +0200
@@ -8,7 +8,7 @@
 CC = $(CROSS_COMPILE)gcc
 CXX = $(CROSS_COMPILE)g++
 PKG_CONFIG = $(CROSS_COMPILE)pkg-config
-CFLAGS := $(shell dpkg-buildflags --get CFLAGS) -Wall
+CFLAGS := $(shell dpkg-buildflags --get CFLAGS) -Wall -std=gnu11
 CPPFLAGS := $(shell dpkg-buildflags --get CPPFLAGS) \
 	-I$(top_srcdir)/$(OUTDIR) \
 	-I$(top_srcdir)/debian/build/build-tools/$(OUTDIR) \
--- a/debian/rules.real	2025-10-20 14:02:29.849384711 +0200
+++ b/debian/rules.real	2025-10-20 16:55:51.906323116 +0200
@@ -41,7 +41,7 @@
 setup_env += KBUILD_VERBOSE=$(if $(filter terse,$(DEB_BUILD_OPTIONS)),0,1)
 
 # XXX: All the tools leak flags between host and build all the time, just don't care.  See #1050991.
-MAKE_CLEAN = $(setup_env) $(MAKE) KCFLAGS=-fdebug-prefix-map=$(CURDIR)/= KBUILD_HOSTCFLAGS='' HOSTCFLAGS='' KBUILD_HOSTLDFLAGS=''
+MAKE_CLEAN = $(setup_env) $(MAKE) KCFLAGS=-fdebug-prefix-map=$(CURDIR)/= KBUILD_HOSTCFLAGS='' HOSTCFLAGS='' KBUILD_HOSTLDFLAGS='' CFLAGS='-std=gnu11'
 MAKE_SELF := $(MAKE) -f debian/rules.real $(MAKEOVERRIDES)
 MAKEOVERRIDES =
 
@@ -350,7 +350,7 @@
 $(STAMPS_DIR)/build_libc-dev:
 	rm -rf '$(DIR)/output'
 	+$(foreach ARCH,$(ALL_LIBCDEV_KERNELARCHES), \
-	   $(MAKE_CLEAN) O='$(CURDIR)/$(DIR)' headers_install ARCH=$(ARCH) INSTALL_HDR_PATH='output/usr'; \
+	   $(MAKE_CLEAN) O='$(CURDIR)/$(DIR)' headers_install ARCH=$(ARCH) INSTALL_HDR_PATH='output/usr' CFLAGS='$(CFLAGS) -std=gnu11'; \
 	   mkdir -p '$(DIR)/output/usr/lib/linux/uapi/$(ARCH)'; \
 	   mv '$(DIR)/output/usr/include/asm' '$(DIR)/output/usr/lib/linux/uapi/$(ARCH)/asm'; )
 	@$(stamp)
