--- a/debian/patches/series	2025-09-08 10:15:48.032750291 +0200
+++ b/debian/patches/series	2025-09-08 10:17:12.264801000 +0200
@@ -75,10 +75,10 @@
 bugfix/all/disable-some-marvell-phys.patch
 bugfix/all/fs-add-module_softdep-declarations-for-hard-coded-cr.patch
 bugfix/all/documentation-use-relative-source-paths-in-abi-documentation.patch
-bugfix/all/ext4-don-t-try-to-clear-the-orphan_present-feature-b.patch
+# bugfix/all/ext4-don-t-try-to-clear-the-orphan_present-feature-b.patch # already in 6.12.44
-bugfix/all/alloc_fdtable-change-calling-conventions.patch
-bugfix/all/net-ipv4-fix-regression-in-local-broadcast-routes.patch
+# bugfix/all/alloc_fdtable-change-calling-conventions.patch # already in 6.12.44
+# bugfix/all/net-ipv4-fix-regression-in-local-broadcast-routes.patch # already in 6.12.45
 
 # Miscellaneous features
