--- a/debian/patches/series	2025-09-10 12:02:09.721924979 +0000
+++ b/debian/patches/series	2025-09-10 14:50:59.976526194 +0000
@@ -75,10 +75,6 @@
 bugfix/all/disable-some-marvell-phys.patch
 bugfix/all/fs-add-module_softdep-declarations-for-hard-coded-cr.patch
 bugfix/all/documentation-use-relative-source-paths-in-abi-documentation.patch
-bugfix/all/ext4-don-t-try-to-clear-the-orphan_present-feature-b.patch
-bugfix/all/proc-fix-missing-pde_set_flags-for-net-proc-files.patch
-bugfix/all/alloc_fdtable-change-calling-conventions.patch
-bugfix/all/net-ipv4-fix-regression-in-local-broadcast-routes.patch

 # Miscellaneous features
